common-data/src/main/scala/org/genivi/sota/data/Interval.scala

```diff
@@ -0,0 +1,22 @@
+package org.genivi.sota.data
+
+import java.time.Instant
+
+case class Interval(start: Instant, end: Instant) {
```

Comment:
Two places:
```
package org.genivi.sota.core.data
case class UpdateRequest
  periodOfValidity: Interval,
```
and
```
package org.genivi.sota.core.data.client
case class ClientUpdateRequest
```

Field `UpdateRequest.periodOfValidity` isn't used, at least not currently.